src/scripts/Events/EventsList.js

```diff
@@ -0,0 +1,27 @@
+import { getEvents, useEvents } from "./EventDataProvider.js"
+import {eventsHTML} from "./Events.js"
+
+
+
+
+const contentTarget = document.querySelector(".contentContainer")
```

Comment:
The EventsList contentTarget should be ".eventsContainer" within the contentContainer I think?